Jenkinsfile.native

```
@@ -0,0 +1,109 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
+pipeline {
+    agent {
+        label 'kie-rhel7 && kie-mem16g'
```

As mentioned by @cristianonicolai 

```suggestion
        label 'kogito-cloud'
```

---

`kogito-cloud` has 16GB mem already

---

Done